libweb3jsonrpc/WebThreeStubServerBase.cpp

```diff
@@ -174,10 +178,14 @@ string WebThreeStubServerBase::eth_getBlockTransactionCountByHash(string const&
 	}
 }
 
-string WebThreeStubServerBase::eth_getBlockTransactionCountByNumber(string const& _blockNumber)
+Json::Value WebThreeStubServerBase::eth_getBlockTransactionCountByNumber(string const& _blockNumber)
 {
 	try
 	{
+		BlockNumber blockNumber = jsToBlockNumber(_blockNumber);
+		if (!client()->isKnown(blockNumber))
+			return Json::Value(Json::nullValue);
+
```

Comment:
This might be a bit off, but can it happen that between line 186 and 189, we switch to a new fork which has a smaller block number count (but higher total difficulty) such that the block number is not valid anymore?